src/tensoroperations.jl

```diff
@@ -1,54 +1,55 @@
 # TensorOperations compatiblity
 #-------------------------------
-function tensoradd!(C::SparseArray{<:Any,N}, indCinA,
-                    A::SparseArray{<:Any,N}, CA::Symbol,
+function tensoradd!(C::SparseArray{<:Any,N},
+                    A::SparseArray{<:Any,N}, conjA::Bool, pA,
                     α::Number=One(), β::Number=One()) where {N}
-    (N == length(indCinA) && TupleTools.isperm(indCinA)) ||
-        throw(ArgumentError("Invalid permutation of length $N: $indCinA"))
-    size(C) == TupleTools.getindices(size(A), indCinA) ||
+    (N == length(pA) && TupleTools.isperm(pA)) ||
+        throw(ArgumentError("Invalid permutation of length $N: $pA"))
+    size(C) == TupleTools.getindices(size(A), pA) ||
         throw(DimensionMismatch("non-matching sizes while adding arrays"))
```

Comment:
```suggestion
    argcheck_tensoradd(C, A, pA)
```

Reply:
This only works after making TensorOperations a dependency, right? Also, I currently do not yet use `Index2Tuple` in the arguments for these methods. I will first study your PR on top of this one.